cli/flox/src/commands/mod.rs

```diff
@@ -205,6 +205,9 @@ impl FloxArgs {
 
         if !config.flox.disable_metrics {
             init_telemetry(&config.flox.data_dir, &config.flox.cache_dir).await?;
+            let connection = AWSDatalakeConnection::default();
```

Comment:
nonblocking: why isn't this done inside `init_telemetry`?

Reply:
`init_telemetry` is just initializing the telemetry uuid. I updated the method name to better reflect that.